lib/source-destination/configured-source/configured-source.ts

```diff
@@ -264,10 +264,8 @@ export class ConfiguredSource extends SourceSource {
 		}
 		for (const partition of partitions) {
 			try {
-				await interact(this.disk, partition.index, async (fs) => {
-					// @ts-ignore: trim method exists for ext partitions
+				await interact(this.disk, partition.index, async (fs: any) => {
```

Comment:
@aethernet did we have to use `any` here?
Replacing `@ts-ignore` with `@ts-expect` error along with the comment of why we had to do this would be way better.

Reply:
https://github.com/balena-io-modules/etcher-sdk/pull/310